torchsde/_brownian/modified.py

```diff
@@ -46,21 +46,22 @@ def levy_area_approximation(self):
 # TODO: these checks are very inelegant. Is there a better interface to Brownian motion?
 class ReverseBrownian(_ModifiedBrownian):
     def __call__(self, ta, tb, return_U=False, return_A=False):
-        # TODO: double-check if U and A need reversing
+        # Whether or not to negate the statistics depends on the return value of the adjoint SDE. Currently, the adjoint
+        # returns negated drift and diffusion, so we don't negate here.
         if return_U:
             if return_A:
                 W, U, A = self.base_brownian(-tb, -ta, return_U=return_U, return_A=return_A)
-                return tuple(-W_ for W_ in W), U, A
+                return W, U, A
```

Comment:
This looks wrong to me. By the time it gets called in ReversedBrownian then it should already be wrapped in a TupleBrownian.

Reply:
Actually, looking at it now: it only gets wrapped into a `TupleBrownian` if there's tensor (not tuple) input for the forward pass. I'm guessing you're running into issues with tupled input?

For now we could always wrap into `TupleBrownian`, but frankly this whole tensor/tuple thing is a complete headache that may need sorting sooner rather than later.

Reply:
This file is a bit outdated, and you may check the latest. The issue is mostly regarding whether or not we should negate the sign. 

On another note, I agree the tuple issue is annoying and agree to remove this if I have additional bandwidth.